src/cantools/database/can/formats/dbc.py

```diff
@@ -169,6 +169,22 @@
     minimum=0,
     maximum=2**16-1)
 
+ATTRIBUTE_DEFINITION_GENMSGDELAYTIME = AttributeDefinition(
+    name='GenMsgDelayTime',
+    default_value=0,
+    kind='BO_',
+    type_name='INT',
+    minimum=0,
+    maximum=2**16-1)
+
+ATTRIBUTE_DEFINITION_GENMSGSTARTDELAYTIME = AttributeDefinition(
+    name='GenMsgStartDelayTime',
+    default_value=0,
+    kind='BO_',
+    type_name='INT',
+    minimum=0,
+    maximum=2**16-1)
+
```

Comment:
Missing `ATTRIBUTE_DEFINITION_GENMSGTYPE` (even if w/o default value, or other specifications).
If you don't have all the details but you think these details should be added after further investigation, let's add a TODO to the fields of the attribute (ENUM choiches I guess) that should be extended.

Reply:
added the TODO for now, we can check later on which is the correct default type for the `ATTRIBUTE_DEFINITION_TYPE` (as far as I see in the DBC files, the GENMSG prefix is absent)